src/meta-srv/src/service/admin/node_lease.rs

```diff
@@ -34,11 +34,9 @@ impl HttpHandler for NodeLeaseHandler {
         &self,
         _: &str,
         _: http::Method,
-        params: &HashMap<String, String>,
+        _params: &HashMap<String, String>,
```

Comment:
```suggestion
```

Reply:
> 

We cannot simply remove this parameter since it's used here:
https://github.com/GreptimeTeam/greptimedb/blob/4045298cb2b39ca8dbc98d4c25bafa3fc910bb25/src/meta-srv/src/service/admin/heartbeat.rs#L37